blueocean-rest/README.md

```diff
@@ -110,18 +110,17 @@ To avoid needing a crumb to POST data, the header `Content-Type: application/jso
 
 # Security
 
+NOTE: JWT is disabled by default for now. to enable JWT authentication use FEATURE_BLUEOCEAN_JWT_AUTHENTICATION=true system property.
```

Comment:
Semi-standard way is to use the fully qualified class name dot property name, which is a reasonable thing to do.